core/src/main/java/com/google/instrumentation/trace/TraceOptions.java

```diff
@@ -183,7 +183,7 @@ private Builder(byte options) {
      *
      * @return this.
      */
-    Builder setIsSampled() {
+    public Builder setIsSampled() {
```

Comment:
one thing I like is to make an Internal class which makes it obvious that this is not exposed for others to use.

Ex. Internal.setIsSampled(options);

you can see examples of this in Brave and okhttp